x-pack/legacy/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/apm/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+import React, { Fragment } from 'react';
+import {
+  EuiSpacer,
+  EuiCodeBlock,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiCallOut,
+  EuiText
+} from '@elastic/eui';
+import { Monospace } from '../components/monospace';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { statusTitle } from './common_apm_instructions';
+import { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } from 'ui/documentation_links';
+
+export function getApmInstructionsForEnablingMetricbeat(product, _meta, {
+  esMonitoringUrl,
+  hasCheckedStatus,
+  checkingMigrationStatus,
+  checkForMigrationStatus,
+  autoCheckIntervalInMs
+}) {
+  const securitySetup = (
+    <Fragment>
+      <EuiSpacer size="m"/>
+      <EuiCallOut
+        color="warning"
+        iconType="help"
+        title={(
+          <EuiText>
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.apmInstructions.metricbeatSecuritySetup"
+              defaultMessage="If security features are enabled, there may be more setup required.{link}"
```

Comment:
Looks like you might need a space after the dot.

Reply:
There is a space in the link fragment actually that should get the job done, unless you're not seeing a space when running it?

Reply:
Was just looking from the code point of view, but I understand now. Will test in a little bit 👍